apps/transport/lib/jobs/resource_unavailable_job.ex

```diff
@@ -118,12 +88,23 @@ defmodule Transport.Jobs.ResourceUnavailableJob do
     payload
   end
 
-  defp update_availability({is_available, %Resource{} = resource}) do
+  # We’ve just updated the URL by following it until we got a 200, so it’s available
+  defp check_availability({:updated, %Resource{} = resource}) do
+    {true, resource}
+  end
+
+  defp check_availability({:no_op, %Resource{format: format} = resource}) do
+    download_url = Resource.download_url(resource)
+    is_available = Transport.AvailabilityChecker.Wrapper.available?(format, download_url)
+    {is_available, resource}
+  end
+
+  defp update_resource({is_available, %Resource{} = resource}) do
     resource |> Resource.changeset(%{is_available: is_available}) |> DB.Repo.update!()
```

Comment:
```suggestion
    resource = resource |> Resource.changeset(%{is_available: is_available}) |> DB.Repo.update!()
```
Sinon la ressource passée plus loin n'a pas le champ `is_available` modifié. Surement pas grave avec le code actuel